test/e2e/utils.go

```diff
@@ -215,10 +215,17 @@ func deleteNodeNeworkStates() {
 }
 
 func deletePolicy(name string) {
+	By(fmt.Sprintf("Deleting policy %s", name))
 	policy := &nmstatev1alpha1.NodeNetworkConfigurationPolicy{}
 	policy.Name = name
 	err := framework.Global.Client.Delete(context.TODO(), policy)
 	Expect(err).ToNot(HaveOccurred())
+
+	policy = nil
```

Comment:
we don't need this, you can `Get` directly into anonymous `&nmstatev1alpha1.NodeNetworkConfigurationPolicy{}`.